rero_mef/agents/mef/views.py

```diff
@@ -44,3 +47,65 @@ def redirect_item(pid):
             'invenio_records_rest.mef_item', pid_value=pid, **request.args),
         code=308
     )
+
+
+api_blueprint_latest = Blueprint(
+    'api_agents_mef_latest',
+    __name__,
+    url_prefix='/agents/mef'
+)
+
+
+@api_blueprint_latest.route('/latest/<pid_type>:<pid>')
+def get_latest(pid_type, pid):
+    """Get latest MEF for pid and type."""
+    from rero_mef.agents.mef.api import AgentMefRecord
+    return jsonify(AgentMefRecord.get_latest(pid_type=pid_type, pid=pid))
+
+
+@api_blueprint_latest.route('/updated/', methods=['POST'])
```

Comment:
POST really ? POST request are used to send data to the application (in REST for create resource - PUT to update)

Reply:
We have to use POST because there could be a lot of pids to filter.